src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -122,9 +121,9 @@ export class ArtworkApp extends React.Component<Props> {
   }
 
   renderArtists() {
-    const artists = get(this.props, p => p.artwork.artists)
+    const artists = this.props.artwork?.artists
 
-    if (!artists.length) {
+    if (!artists?.length) {
```

Comment:
Fixes https://sentry.io/organizations/artsynet/issues/1529777601/?project=28316&query=is%3Aunresolved&statsPeriod=24h

Reply:
Nice. I was unsure of what `artist?.length` would be if `artist` was undefined and it looks like the answer to that is... well.. `undefined`.

👍 Love that syntax though.